### ISSUE TITLE
#MediaBrowser.Backdrop not set in UserView

### ISSUE THREAD

**Issue by ncoH** [apondman/MediaBrowser#54]
The backdrop is only visible in UserRootFolder

**Comment by apondman** [comment]
what is the desired effect here? I don't think UserViews have backdrops currently.
I could however show a parent backdrop perhaps.

**Comment by ncoH** [comment]
Yes, there is no backdrop for user view but it should show the same as the parent root folder.
